resolwe_bio/tests/test_variant_calling.py

```diff
@@ -59,7 +59,7 @@ def test_variant_calling_gatk_joint(self):
         self.run_processor('vc-gatk-joint', inputs)
         # NOTE: output can not be tested
 
-    @unittest.skip("Missing tools in runtime")
+    @unittest.skipIf(running_on_old_platform, "Missing tools in runtime")
```

Comment:
I've added Cheng Lin's tool in ansible, so you probably just need to past instructions from Janez's mail to ``provision-already_in_ansible``.